src/stubs/extensions/qml/MuseScore/Extensions/ExtensionsPage.qml

```diff
@@ -0,0 +1,7 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+
+Item {
+    property string search: ""
+    property string backgroundColor: ""
+}
```

Comment:
I think better to add Text like "Extensions Page Stub"

Reply:
added